config/config.go

```diff
@@ -117,6 +117,7 @@ type Log struct {
 	SlowThreshold      uint64 `toml:"slow-threshold" json:"slow-threshold"`
 	ExpensiveThreshold uint   `toml:"expensive-threshold" json:"expensive-threshold"`
 	QueryLogMaxLen     uint64 `toml:"query-log-max-len" json:"query-log-max-len"`
+	SlowLogPlan        uint32 `toml:"slow-log-plan" json:"slow-log-plan"`
```

Comment:
slow-log-output-plan

Reply:
How about `show-plan-in-slow-log`? Or we can add a `slow-log` section to the config, put all the configurations related to slow log into that section. For example, query time threshold, log file, whether to output plan, etc.

Reply:
Great, maybe we can refactor this code later, I will just change the config name in this PR.